notes-service/src/main/java/org/exoplatform/wiki/service/rest/NotesRestService.java

```diff
@@ -539,6 +539,8 @@ public Response updateNoteById(@ApiParam(value = "Note id", required = true) @Pa
           WikiPageParams noteParams = new WikiPageParams(note_.getWikiType(), note_.getWikiOwner(), newNoteName);
           noteService.removeDraftOfNote(noteParams);
         }
+      } else if (note_.getContent().equals(note.getContent())) {
```

Comment:
This modification will launch listeners related to edit content event, for example update the last modification date. 

If the content is not modified we should note update the note.
Can you try to find another solution ?

Thanks